darwin/DarwinProcess.c

```diff
@@ -269,6 +269,20 @@ static void DarwinProcess_updateCmdLine(const struct kinfo_proc* k, Process* pro
    Process_updateCmdline(proc, k->kp_proc.p_comm, 0, strlen(k->kp_proc.p_comm));
 }
 
+// Converts ticks in the Mach "timebase" to nanoseconds.
+// See `mach_timebase_info`, as used to define the `Platform_timebaseToNS` constant.
+static uint64_t machTicksToNanoseconds(uint64_t schedulerTicks) {
+   double nanoseconds_per_mach_tick = Platform_timebaseToNS;
+   return (uint64_t) (nanoseconds_per_mach_tick * (double) schedulerTicks);
+}
```

Comment:
Similar to `Platform_clockTicksPerSec`, this conversion function is the only usage of `Platform_timebaseToNS`. I suggest we refactor:

1. Hide the `Platform_timebaseToNS` constant from `Platform.h`
2. Move `machTicksToNanoseconds` to `Platform.c`
3. Expose `machTicksToNanoseconds` from `Platform.h`, and call that from here.

Reply:
@BenBE Here's what the looks like https://github.com/amomchilov/htop/pull/1/files

Once this PR ships, I'll open a proper PR on this repo for that.

Reply:
> Similar to `Platform_clockTicksPerSec`, this conversion function is the only usage of `Platform_timebaseToNS`. I suggest we refactor:
> 
>     1. Hide the `Platform_timebaseToNS` constant from `Platform.h`
> 
>     2. Move `machTicksToNanoseconds` to `Platform.c`
> 
>     3. Expose `machTicksToNanoseconds` from `Platform.h`, and call that from here.

Done: https://github.com/htop-dev/htop/pull/766